packages/open-next/src/build.ts

```diff
@@ -208,8 +208,8 @@ function printOpenNextVersion() {
   logger.info(`OpenNext v${openNextVersion}`);
 }
 
-function initOutputDir() {
-  const { outputDir, tempDir } = options;
+function initOutputDir(tempDir: string) {
```

Comment:
We probably need some comment here as to why we just don't use the tempDir in options